docker/build.sh

```
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-$(mops toolchain bin moc) src/main.mo -o out.wasm -c --debug --public-metadata candid:service --public-metadata candid:args $(mops sources)
+$(mops toolchain bin moc) src/main.mo -o out.wasm -c --debug --public-metadata candid:service --public-metadata candid:args --max-stable-pages 131072 $(mops sources)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)